Sources/MapboxNavigation/JunctionView.swift

```diff
@@ -71,9 +71,6 @@ public class JunctionView: UIImageView {
                     DispatchQueue.main.async {
                         guard let self = self else { return }
                         
-                        self.isCurrentlyVisible = true
```

Comment:
Fixes an issue with presenting junction view when it cannot be presented via `show(animated:)` method.